tests/ClassTest.php

```diff
@@ -464,7 +464,9 @@ class T extends \PHPUnit\Framework\TestCase {
             ],
             'classAliasNoException' => [
                 'code' => '<?php
-                    class_alias("Bar\F1", "Bar\F2");
+                    namespace {
```

Comment:
A namespace declarations must be the first thing in the file. php-parser 5.0 enforces this.

Reply:
This is simply invalid PHP code

https://3v4l.org/9HAnR